curvefs/src/client/s3/disk_cache_write.cpp

```diff
@@ -139,7 +146,7 @@ int DiskCacheWrite::AsyncUploadFunc() {
         std::list<std::string>::iterator iter;
         int ret;
         for (iter = toUpload.begin(); iter != toUpload.end(); iter++) {
-            ret = UploadAndRemove(*iter);
+            ret = UploadFile(*iter);
```

Comment:
Why not remove? When will it be deleted？

Reply:
> Why not remove? When will it be deleted？

delete in aync callback.